code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -7,7 +7,7 @@ GLOBAL_LIST_EMPTY(scp173s)
 
 /mob/living/scp_173
 	name = "SCP-173"
-	desc = "A statue, constructed from concrete and rebar with traces of Krylon brand spray paint"
+	desc = "A statue, constructed from concrete and rebar with traces of Krylon brand spray paint."
```

Comment:
the spellcheck isnt in the changelog fix it

Reply:
suffer.